### ISSUE TITLE
Call my boss

### ISSUE THREAD

**Issue by neutraltone** [aethant/vanilla-css#13]
Tell him I won't be in tomorrow because I'm too busy working on this vanilla CSS project. Tell him it's going to change the world as he knows it.

**Comment by hectorpalmatellez** [comment]
No phone number, not enough information to complete this issue.
:-1:

**Comment by aethant** [comment]
Ok

**Comment by neutraltone** [comment]
Will update if I miss the bus, otherwise I'll close the issue and file as `won't fix`

**Comment by jxm262** [comment]
I vote to leave this open for a somewhat indefinite period of time.  There's a slightly high probability... >20% that I might also need to call my boss for this issue as well.

**Comment by neutraltone** [comment]
Right, update. No need to call my boss, I think I will make the morning meeting.

However... I at with @jxm262 on this one. Could be a recurring issue until someone fixes the root cause of the problem. Any suggestions welcome.

**Comment by Seikho** [comment]
This could be solved by adding IoT into the mix. This should add almost no additional technical debt to the project.

**Comment by aethant** [comment]
Can I close this? Did someone call his boss?

**Comment by hectorpalmatellez** [comment]
My guess is he didn't miss his bus, so he should have closed it.
@neutraltone :question:

**Comment by neutraltone** [comment]
My bad. I made it - it was awful.

**Comment by Seikho** [comment]
I wonder if we should leave this open and add this to the next VanillaCSS Foundation meeting?

**Comment by jxm262** [comment]
Using my power as project collaborator to veto this close and reopen the ticket.  This issue is still not resolved and can be reproduced by visiting my desk at work.  Confirmed I am still here and not at home.  

Keeping open until next week at _least_

**Comment by Seikho** [comment]
This difficult to reproduce. Can we label as intermittent? I'm unable to
visit my desk at all times. There seems to be a pattern. Needs  further
investigation.

**Comment by neutraltone** [comment]
@hectorpalmatellez, @jxm262, @Seikho, @aethant:

Might have to call and issue #13 on tomorrow's activities. Will keep you all posted.

**Comment by jxm262** [comment]
@neutraltone  that's perfectly fine.  Just keep in mind this issue is still relevant to many of us working on this app.  Feel free to work on #13 tomorrow, but please leave the ticket open.  I know I haven't been too active here recently, but the reason for my recent absence directly relates to this specific issue (overworked, can't do this ticket myself yet). 

Please let us know if you make any progress on this.

**Comment by neutraltone** [comment]
OK I might have to reinstate this. I fell into a boozer this evening and subsequently ended up several pitchers deep. It's OK I managed to get home but at which point a bottle Casillero del Diablo (CC:@hectorpalmatellez) forced itself upon me. There was nothing I could do. It made me watch The Sixth Sense.

I think I might be OK, I've managed to convince Diablo to go back into the cupboard but who know's when he'll surface again. Please keep this issue in mind.

Help a brother out.

**Comment by hectorpalmatellez** [comment]
Siempre debes respetar al diablo y su casillero, @neutraltone.

Your boss will understand, I'm sure.

**Comment by neutraltone** [comment]
Sorry to resurrect but I've had to many @Hectorpalmatellez's, I think it might be unmerable...

**Comment by zivc** [comment]
> Casillero del Diablo

👎  desperate times, off-licence favourite though. Usually end up throwing an exception in the toilet.

**Comment by JDDoesDev** [comment]
This seems to be a recurring case of rectal glaucoma.  Not a code based issue.

**Comment by Seikho** [comment]
Removing `wontfix` label due to the `help wanted` and `wontfix` label paradox.

**Comment by neutraltone** [comment]
Did anyone call my boss? I don't work for him anymore but feel free to phone and call him a cheapskate.

**Comment by aethant** [comment]
Closing. Will reopen when issue resurfaces.

**Comment by Seikho** [comment]
Him and his team of clairvoyants already knew you were going to post that comment, @neutraltone. Notifying him will be unnecessary.

**Comment by neutraltone** [comment]
Recent activities in the application have cause issue to resurface. Reopening so we can get to the bottom of this.

**Comment by neutraltone** [comment]
@aethant can you call my boss?

**Comment by Seikho** [comment]
> @aethant can you call my boss?

Does the css api support this?

**Comment by aethant** [comment]
> @aethant can you call my boss?

I don't even know where you work, mate. Just have another bitter and lean into it. I'm sure it'll be fine.

**Comment by aethant** [comment]
> > @aethant can you call my boss?
> 
> 
> 
> Does the css api support this?
> 
> 

Theoretically, yes.

```
.sobriety {
   display: none;
}
```

**Comment by Seikho** [comment]
I feel like the CSS api is too powerful. Is the world ready for something like this?
Have we gone too far?

**Comment by lmosele** [comment]
Hi has anyone called @neutraltone's boss yet? We're adding this to our codebase and need this resolved
